### PR TITLE
fix(example): Fix example.py bot

### PR DESCRIPTION
**Pull request by Nanrech**
## About

This pull request fixes an incorrect event listener and an incorrect statement about whether a line will or won't execute.

## Checklist

- [x] I've ran `pre-commit` to format and lint the change(s) made.
- [x] I've checked to make sure the change(s) work on `3.8.6` and higher.
- [x] This fixes/solves an [Issue](https://github.com/goverfl0w/discord-interactions/issues).
  - #712
- [ ] I've made this pull request for/as: (check all that apply)
  - [x] Documentation
  - [ ] Breaking change
  - [ ] New feature/enhancement
  - [x] Bugfix

### CODE REVIEW

**Comment by r-warfield** [comment]
am i missing something? you do realize its possible to make event callbacks by providing the name to the decorator right?

**Comment by Nanrech** [comment]
> am i missing something? you do realize its possible to make event callbacks by providing the name to the decorator right?

It wasn't when I created the PR.

**Comment by sillyElvy** [comment]
Its broken lol

You should be able to do it via `name="name"` in the decorator on the unstable branch

**Comment by Nanrech** [comment]
Yep, should I change it then now or do I wait until that change is released?

**Comment by i0bs** [comment]
If it's already on unstable, you can make the change now

**Comment by Nanrech** [comment]
Updated, should be fine now.

File: examples/bot.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 # With our client established, let's have the library inform us when the client is ready.
 # These are known as event listeners. An event listener can be established in one of two ways.
 # You can provide the name of the event, prefixed by an "on_", or by telling the event decorator what event it is.
-@client.event
+@client.event(name="on_ready")
```
Comment (sillyElvy) [inline examples/bot.py]: You don't need name if the function name is the events name

**Review by sillyElvy — CHANGES_REQUESTED** [review]

File: examples/bot.py (hunk 1 above)
Comment (Nanrech) [reply examples/bot.py]: Removed the name param

**Review by sillyElvy — APPROVED** [review]

**Pull request merged**